adhocracy4/images/validators.py

```diff
@@ -74,3 +76,21 @@ def validate_image(
     if errors:
         raise ValidationError(errors)
     return image
+
+
+@deconstructible
+class ImageAltTextValidator:
+    """Validate that if the input contains html img tags that all have the alt
+    attribute set, otherwise raise ValidationError.
+    """
```

Comment:
It needs attributes `message`, `code`, so the user knows what to fix (e.g alt in img tag is missing) , then adding these in the `__eq__` function. And is missing an `__init__`.
The inline docs better be placed under the call function.

Reply:
what purpose does the code have? I can't really figure it out. And as code and messages are always the same, do we need to check for them in `__eq__` ? They will always match.

Reply:
code is `invalid`.
You can see how they do it in [django valiators](https://github.com/django/django/blob/main/django/core/validators.py)